azure/converters/identity.go

```diff
@@ -59,7 +59,7 @@ func UserAssignedIdentitiesToVMSSSDK(identities []infrav1.UserAssignedIdentity)
 	return userIdentitiesMap, nil
 }
 
-// sanitized removes "azure:///" prefix from the given id
+// sanitized removes "azure://" prefix from the given id
 func sanitized(id string) string {
-	return strings.TrimPrefix(id, "azure:///")
+	return strings.TrimPrefix(id, "azure://")
```

Comment:
can we create a util method for this since we are using it in another place as well? we could probably include one for creating the id as well so that it's all in one place. wdyt?